File: 2418_Sort_the_People.js
var sortPeople = function(names, heights) {
    let ans = []
    for(let i = 0; i < names.length; i++){
        let max_index = heights.indexOf(Math.max(...heights))
        ans.push(names[max_index])
        heights[max_index] = -1
    }
    return ans
};

File: 2418_Sort_the_People.cpp
class Solution {
public:
    vector<string> sortPeople(vector<string>& names, vector<int>& heights) {
        unordered_map<int, string> mp;
        int i = 0;
        while(i < names.size()){
            mp[heights[i]] = names[i];
            i++;
        }
        sort(heights.begin(), heights.end(), greater<>());
        i = 0;
        while(i < names.size()){
            names[i] = mp[heights[i]];
            i++;
        }            
        return names;
    }
};
